From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Thu, 21 Aug 2014 12:30:15 +0200
Subject: [PATCH] missing: add BPF_XOR

BPF_XOR was introduced in kernel 3.7

Not for upstream, but needed for host-systemd.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 src/basic/missing.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/basic/missing.h b/src/basic/missing.h
index e5554c6e1a22..5f7d9c65d816 100644
--- a/src/basic/missing.h
+++ b/src/basic/missing.h
@@ -898,6 +898,10 @@ struct btrfs_ioctl_quota_ctl_args {
 #  define BPF_XOR 0xa0
 #endif
 
+#ifndef BPF_XOR
+#  define BPF_XOR 0xa0
+#endif
+
 /* Note that LOOPBACK_IFINDEX is currently not exported by the
  * kernel/glibc, but hardcoded internally by the kernel.  However, as
  * it is exported to userspace indirectly via rtnetlink and the
